olive/passes/olive_pass.py

```diff
@@ -80,7 +81,7 @@ def __init__(
         if self._requires_user_script:
             self._user_module_loader = UserModuleLoader(self._config["user_script"], self._config["script_dir"])
         if self._requires_data_config:
-            data_config = self._config["data_config"] or {}
+            data_config = self._config.get("data_config") or {}
```

Comment:
get("data_config", {})

Reply:
`data_config` is in self._config but the value is None. So the logic here is like:
```python
if 'data_config' not in self._config or self._config['data_config'] is None:
    data_config = {}
```

Reply:
then the original line seems ok. we don't need change it. right?